api/backend.go

```diff
@@ -605,11 +636,13 @@ func (b *StatusBackend) startBrowsers(password string) error {
 	if err != nil {
 		return err
 	}
-	account, err := b.accountManager.SelectedWalletAccount()
+
+	mainAccountAddress, err := b.accountManager.MainAccountAddress()
 	if err != nil {
 		return err
 	}
-	path := path.Join(b.statusNode.Config().DataDir, fmt.Sprintf("browsers-%x.sql", account.Address))
+
+	path := path.Join(b.statusNode.Config().DataDir, fmt.Sprintf("browsers-%x.sql", mainAccountAddress))
```

Comment:
@gravityblast the reason i was asking if `mainAccountAddress` can change later is that if it will then new db will be instantiated, which is quite a bad thing :)

Reply:
@dshulyak you are right! it doesn't now, but it will be possible to change it in the future. 
So maybe in your PR #1530 we can add a unique ID and use that one instead of mainAccountAddress?

Reply:
sounds good